charts/budibase/templates/app-service-deployment.yaml

```diff
@@ -24,12 +21,9 @@ spec:
   template:
     metadata:
       annotations:
-        kompose.cmd: kompose convert
-        kompose.version: 1.21.0 (992df58d8)
 {{ if .Values.services.apps.templateAnnotations }}
 {{- toYaml .Values.services.apps.templateAnnotations | indent 8 -}}
 {{ end }}
-      creationTimestamp: null
```

Comment:
I believe this was here to mitigate the fact that our images didn't update their tag for each new version, so we needed a way to trick helm into refetching the image. Is the tagging issue resolved now?

Reply:
We have an `imagePullPolicy` of `Always`, so I don't think this trick is necessary.